crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip.rs

```diff
@@ -55,7 +57,7 @@ pub(crate) mod tests;
 /// max send buffer size (keep it under 16384 with a little room for overhead)
 /// (this is not a tuning_param because it must be coordinated
 /// with the constant in PoolBuf which cannot be set at runtime)
-const MAX_SEND_BUF_BYTES: usize = 16000;
+const MAX_SEND_BUF_BYTES: usize = 16_000_000;
```

Comment:
```suggestion
/// ^^ obviously we're no longer following the above advice..
///    in the case of the pool buf management, any gossips larger than
///    16000 will now be shrunk resulting in additional memory thrashing
const MAX_SEND_BUF_BYTES: usize = 16_000_000;
```

Reply:
If this is a problem I can shrink the chunks and reassemble on the other side?